README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://rudderstack.com/">
-    <img src="https://user-images.githubusercontent.com/59817155/121357083-1c571300-c94f-11eb-8cc7-ce6df13855c9.png">
```

Comment:
I use Github's dark theme. It is difficult to see the logo in that.
Attaching the older one and the new one for reference

<img width="1150" alt="Screenshot 2022-01-06 at 9 14 02 AM" src="https://user-images.githubusercontent.com/7383674/148325231-3639746c-ce3a-46d7-80d8-5b39c41203f5.png">

<img width="1177" alt="Screenshot 2022-01-06 at 9 13 45 AM" src="https://user-images.githubusercontent.com/7383674/148325282-9eb108aa-e7c7-4520-8572-64080534fc5e.png">

Reply:
Done